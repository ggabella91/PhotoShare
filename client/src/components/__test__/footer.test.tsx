import { shallow } from 'enzyme';
import React from 'react';
import Footer from '../footer/footer.component';

it('renders a footer component', () => {
  const footerWrapper = shallow(<Footer />);

  expect(footerWrapper).toMatchSnapshot();
});
