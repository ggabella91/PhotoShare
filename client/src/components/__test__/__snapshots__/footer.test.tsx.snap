// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a footer component 1`] = `
<div>
  <div
    className="footer"
  >
    <span>
      Created by Giuliano Gabella
    </span>
    <a
      className="favicon-link"
      href="https://icons8.com/icon/Z6RmPnIUvybQ/photo-gallery"
    >
      Photo Gallery icon by Icons8
    </a>
  </div>
</div>
`;
